Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM node:7
+WORKDIR /app
+COPY package.json /app
+RUN npm install
+COPY . /app
+CMD node merge.js -o merged.json WES-swagger-template.json http://wf-search.light.overture.bio/v2/api-docs http://wf-management.light.overture.bio/v2/api-docs && node app.js
```

Comment:
running a node process is fine, if the app fails we want the process to fail so kubernetes picks up on that and restarts the pod.

On a different note though, I think the `node merge.js -o merged.json WES-swagger-template.json http://wf-search.light.overture.bio/v2/api-docs http://wf-management.light.overture.bio/v2/api-docs` should probably be part of a `RUN` line rather than built into here, so that the `merged.json` is bundled in the image